test/test-platform-output.c

```diff
@@ -61,13 +62,16 @@ TEST_IMPL(platform_output) {
   ASSERT(rusage.ru_utime.tv_usec >= 0);
   ASSERT(rusage.ru_stime.tv_sec >= 0);
   ASSERT(rusage.ru_stime.tv_usec >= 0);
+  ASSERT(rusage.ru_majflt >= 0);
   printf("uv_getrusage:\n");
   printf("  user: %llu sec %llu microsec\n",
          (unsigned long long) rusage.ru_utime.tv_sec,
          (unsigned long long) rusage.ru_utime.tv_usec);
   printf("  system: %llu sec %llu microsec\n",
          (unsigned long long) rusage.ru_stime.tv_sec,
          (unsigned long long) rusage.ru_stime.tv_usec);
+  printf("  page faults: %" PRIu64 "\n", rusage.ru_majflt);
```

Comment:
`PRIu64` doesn't seem to be used anywhere else in the project. I'm not sure if this is OK, or if you should use `%llu`.